src/dev.html

```diff
@@ -71,6 +70,7 @@
       <link rel="stylesheet" href="/media/css/reviews.styl.css">
       <link rel="stylesheet" href="/media/css/search.styl.css">
       <link rel="stylesheet" href="/media/css/settings-hover-desktop.styl.css">
+      <link rel="stylesheet" href="/media/css/standalone.styl.css">
```

Comment:
not sure what standalone means. Maybe we should just pull the styles into our base CSS.